djangocms_bootstrap4/contrib/bootstrap4_grid/migrations/0004_remove_bootstrap4gridcolumn_column_size.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bootstrap4_grid', '0003_migrate_column_size'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='bootstrap4gridcolumn',
+            name='column_size',
```

Comment:
maybe add a reverse ata migration to copy the values in the other direction again